ledger-service/http-json/src/main/scala/com/digitalasset/http/Endpoints.scala

```diff
@@ -151,27 +138,34 @@ class Endpoints(
         _ => healthService.ready().map(_.toHttpResponse)
     }
     import scalaz.std.partialFunction._, scalaz.syntax.arrow._
-    ((commandDispatch orElse
+    val dispatch = commandDispatch orElse
       queryAllDispatch orElse
       queryMatchingDispatch orElse
       fetchDispatch orElse
       getPartyDispatch orElse
       allocatePartyDispatch orElse
       packageManagementDispatch orElse
-      liveOrHealthDispatch) &&& { case r => r }) andThen { case (lcFhr, req) =>
-      extendWithRequestIdLogCtx(implicit lc => {
-        val t0 = System.nanoTime
-        logger.info(s"Incoming request on ${req.uri} from ${requestSource(req)}")
-        metrics.daml.HttpJsonApi.httpRequestThroughput.mark()
-        for {
-          res <- lcFhr(lc)
-          _ = {
-            logger.trace(s"Processed request after ${System.nanoTime() - t0}ns")
-            logger.info(s"Responding to client with HTTP ${res.status}")
-          }
-        } yield res
-      })
-    }
+      liveOrHealthDispatch
+    dispatch
+      .&&& { case r => r }
```

Comment:
switched to point based here, because the formatter formats the code to a more readable way here